christfellowshipchurch/web-app-v2
# Watch Online" button opens up in new tab CFDP-1366

## Description 
Clicking on the "Watch Online" button on the Navigation Header now opens up in a new tab

## How to Test 
Click on the "Watch Online" button on the Navigation Header and it should open up in a new tab

## Jira Ticket 
[CFDP-1366]

[CFDP-1366]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1366
